### PR TITLE
Update axd.json

### PR DESCRIPTION
**Pull request by alexdhg**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ ] The file is in the `domains` folder and is in the JSON format.
- [ ] You have completed your website. 
- [ ] The website is reachable.  
- [ ] You're not using Vercel or Netlify.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/axd.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
   "owner": {
     "username": "alexdhg",
-    "email": "alex.zr71@gmail.com"
+    "email": "axd@webwarp.ro"
   },
   "record": {
-    "URL": "https://axxd.me"
+    "CNAME": "https://axxd.me"
```
Comment (Stef-00012) [inline domains/axd.json:7]: Invalid CNAME record (can not have `https://`)

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Review by alexdhg — COMMENTED** [review]
done

**Comment by Stef-00012** [comment]
seems good now

**Review by iostpa — APPROVED** [review]
seems good to me

**Pull request merged**